tests/integration/controller/kyma/helper_test.go

```diff
@@ -24,12 +25,24 @@ const (
 )
 
 func RegisterDefaultLifecycleForKyma(kyma *v1beta2.Kyma) {
+	mandatoryTemplate := builder.NewModuleTemplateBuilder().
+		WithModuleName("mandatory-template-operator").
+		WithChannel("mandatory").
```

Comment:
same as what described [here](https://github.com/kyma-project/lifecycle-manager/pull/1173/files#r1431678322), for mandatory template, the channel is an irrelevant field.

If you define `mandatory` channel as a test case, it leaves the impression that this channel is required by mandatory template, what you can do is define some variable which indicate this field can be ignored, like `dummy-channel`.